lib/chef/handler/datadog.rb

```diff
@@ -129,15 +129,19 @@ def restore_env_proxies
       def prepare_the_pack
         dogs = []
         endpoints.each do |url, api_key, app_key|
-          dogs.push(Dogapi::Client.new(
-                      api_key,
-                      app_key,
-                      nil,   # host
-                      nil,   # device
-                      false, # silent
-                      nil,   # timeout
-                      url
-          ))
+          begin
+            dogs.push(Dogapi::Client.new(
+                        api_key,
+                        app_key,
+                        nil,   # host
```

Comment:
Another approach which would allow the run to still succeed would be to set this line to a non falsy value which would make the hostname logic in dogapi-rb from running. https://github.com/DataDog/dogapi-rb/blob/master/lib/dogapi/facade.rb#L77